pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -156,7 +156,7 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 		ClusterDNSIP:        clusterDNSIP,
 		EtcdCertSignerImage: "quay.io/coreos/kube-etcd-signer-server:678cc8e6841e2121ebfdb6e2db568fce290b67d6",
 		EtcdctlImage:        "quay.io/coreos/etcd:v3.2.14",
-		BootkubeImage:       "quay.io/coreos/bootkube:v0.10.0",
+		BootkubeImage:       "docker.io/sttts/bootkube:latest",
```

Comment:
probably didn't intend to commit this?

Reply:
still WIP until we have the new bootkube.

Reply:
> still WIP until we have the new bootkube.

Have our bootkube pulls merged?

Reply:
yes, we have to bring in the new bootkube release into the installer though. Last bootkube release is from July.

Reply:
@rphillips @aaronlevy could you make a new bootkube release?